### DISCUSSION TITLE
Add clickable messages of acceptation/cancellation of tpa

### DISCUSSION THREAD

**Discussion by danechek** [EssentialsX/Essentials#4054] [Ideas and Feature Suggestions] [answered]
The title says it all.  When sending a request for TP, another player is written that he needs to write to the chat to accept/cancel. You can use the new features of Adventure (KyoriPowered) to set clickable messages of acceptaion/cancellation.

**Comment by JRoy** [comment, marked as answer]
Essentials doesn't support components in its messages and doesn't plan to in 2.x
